ygot/render.go

```diff
@@ -427,7 +429,7 @@ func findUpdatedLeaves(leaves map[*path]interface{}, s GoStruct, parent *gnmiPat
 			continue
 		case reflect.Interface:
 			// This is a union value.
-			/*val, err := unionInterfaceValue(fval, false)
+			/*val, err := unwrapUnionInterfaceValue(fval, false)
```

Comment:
Debugging comments?

Reply:
I'm just changing existing debug comments here to match the new function name.

Reply:
Ah, can we remove them completely? I don't think they need to be there.